src/cider/nrepl/middleware/stacktrace.clj

```diff
@@ -68,10 +77,28 @@
   (let [tool-regex #"^clojure\.lang\.Compiler|^clojure\.tools\.nrepl|^cider\."
         tool? #(re-find tool-regex (or (:name %) ""))
         flag  #(if (tool? %)
-                 (update-in % [:flags] (comp set conj) :tooling)
+                 (flag-frame % :tooling)
                  %)]
     (map flag frames)))
 
+(def directory-namespaces
+  (let [paths (-> (System/getProperty "java.class.path")
```

Comment:
`paths` is a bit misleading here - I think we call these `cp-entries` in other places of the code. Or something like this.